disorder-core/src/Disorder/Core/Gen.hs

```diff
@@ -1,26 +1,31 @@
 module Disorder.Core.Gen (
-    vectorOfSize
+    GenSeed(..)
   , chooseSize
-  , maybeGen
-  , genFromMaybe
-  , smaller
-  , oneofSized
-  , listOfSized
+  , genDeterministic
+  , genDeterministic'
   , genEnum
-  , listWithIndex
-  , listOfSizedWithIndex
+  , genFromMaybe
   -- * re-exports from quickcheck-text
   , genValidUtf8
   , genValidUtf81
+  , listOfSized
+  , listOfSizedWithIndex
+  , listWithIndex
+  , maybeGen
+  , oneofSized
+  , smaller
+  -- * re-exports from quickcheck-text
```

Comment:
Sorry not sure what you mean, you say two blocks but they are straight after each other? Is that just one block or am I missing something